DrivewayDentDeletion/Operators/deploy-im.sh

```diff
@@ -70,11 +70,13 @@ data:
     SET AUTHREC PROFILE('BumperOut') PRINCIPAL('app1') OBJTYPE(QUEUE) AUTHADD(BROWSE,GET,INQ,PUT)
     SET AUTHREC PROFILE('CrumpledIn') PRINCIPAL('app1') OBJTYPE(QUEUE) AUTHADD(BROWSE,GET,INQ,PUT)
     SET AUTHREC PROFILE('CrumpledOut') PRINCIPAL('app1') OBJTYPE(QUEUE) AUTHADD(BROWSE,GET,INQ,PUT)
+    SET CHLAUTH('MTLS.SVRCONN') TYPE(SSLPEERMAP) SSLPEER('CN=demos.${IM_NAME},OU=my-team') USERSRC(MAP) MCAUSER('app1') ACTION(REPLACE)
```

Comment:
The CN needs to match that in the certificate below in this file where it has:
```yaml
commonName: ${NAMESPACE}.${IM_NAME}
```
Note in the certificate it's got the namespace, in the SSLPEER it's got `demos`